File: c/series.c
//Find the sum s=1+x+(x^2/2!)+(x^3/3!)+ ... n terms
#include<stdio.h>
#include<math.h>
int main (){
	float n,x;
	printf("Enter x ");
	scanf("%f",&x);
	printf("Enter no of terms ");
	scanf("%f",&n);
	float sum=0,fact=1;
	sum=sum+1;
	for (int i=1;i<=(n-1);i++){
		fact=fact*i;
		sum=sum+(pow(x,i)/fact);
	}
	printf("%.2f",sum);
	return 0;
}

File: java/test/others/Series.java
// Write a program to determine sum of the following series for given value of n:
// 1 - 1/2^2 + 1/3^2 + ... + 1/n^2
import java.util.Scanner;
import java.lang.Math;

class Series {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    System.out.print("Enter n: ");
    int n = sc.nextInt();

    double res = 0;

    for (int i = 1; i <= n; i++) {
      res += Math.pow((-1),  i+1) / Math.pow(i, 2);
    }

    System.out.println("Res is: " + res);
  }
}